### PR TITLE
[Docs] Fix API links

### LINKED ISSUE #15967: Broken API docs links
The following API doc links are broken: <ul><li>https://developers.cloudflare.com/api/operations/vectorize-update-vectorize-index</li><li>https://developers.cloudflare.com/api/operations/radar-get-connection-tampering-summary</li><li>https://developers.cloudflare.com/api/operations/radar-get-connection-tampering-timeseries-group</li></ul>

### PR DESCRIPTION
**Pull request by pedrosousa**
### Summary

Fixes API links in Dev Docs.
Also adds Radar changelog about endpoint changes.
Closes #15967.

### Notes

API reference links don't need to include the hostname (in fact, you'll get a Vale warning if you do), **unless** the YAML file is being directly used elsewhere outside Dev Docs.
Links in changelog RSS feeds (namely in [Radar's feed](https://developers.cloudflare.com/radar/changelog/index.xml)) will always include the hostname, so that they don't get broken outside the Dev Docs context.

Changelog preview:
https://pedro-issue15967-docs-fix-ap.cloudflare-docs-7ou.pages.dev/radar/changelog/

### CODE REVIEW

**Review by meddulla — APPROVED** [review]

File: data/changelogs/radar.yaml [hunk 1]
```diff
@@ -13,84 +20,83 @@ entries:
...
 - publish_date: '2023-10-31'
   title: Add new Layer 3 direction parameter
   description: |-
     * Added a `direction` parameter to all Layer 3 endpoints. Use together with `location` parameter to filter by origin or
-      target location [timeseries groups](https://developers.cloudflare.com/api/operations/radar-get-attacks-layer3-timeseries-group-by-vector).
+      target location [timeseries groups](/api/operations/radar-get-attacks-layer3-timeseries-group-by-vector).
 - publish_date: '2023-09-08'
   title: Add Connection Tampering endpoints
   description: |-
-    * Added Connection Tampering [summary](https://developers.cloudflare.com/api/operations/radar-get-connection-tampering-summary)
```
Comment (lukevalenta) [inline data/changelogs/radar.yaml:38]: As this is the change log, perhaps we should keep the old `connection-tampering` endpoint? Or is this change needed for the link checker?

**Review by lukevalenta — APPROVED** [review]
One non-blocking comment but looks good otherwise. Thanks!

File: data/changelogs/radar.yaml (hunk 1 above)
Comment (pedrosousa) [reply data/changelogs/radar.yaml:38]: We'll have to either update the URLs so the link checker doesn't complain, or use something like monospaced text instead of real links if we want to show the URLs of the old endpoints. My suggestion is that we update the link here, since the previous URLs actually contained `connection-tampering` and should be easy to identify.

**Review by kodster28 — APPROVED** [review]
